otter/models/cass.py

```diff
@@ -206,7 +206,7 @@ def serialize_json_data(data, ver):
 
 _cql_count_for_tenant = (
     'SELECT COUNT(*) FROM {cf} '
-    'WHERE "tenantId" = :tenantId;')
+    'WHERE "tenantId"=:tenantId {deleting};')
```

Comment:
This query is also used to get count of policies and webhooks from `scaling_policies` and `policy_webhooks` respectively which is why there is place for extra clause. If you prefer I could have two separate queries: One with `deleting=false` for groups and another without it for policies and webhooks.